src/app/containers/CpsMetadata/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Metadata from '../Metadata';
+import getBrandedImage from './utils/getBrandedImage';
+
+const CpsMetadata = ({
+  title,
+  language,
+  description,
+  firstPublished,
+  lastPublished,
+  imageLocator,
+  imageAltText,
+}) => {
+  const { service, articleAuthor } = useContext(ServiceContext);
+  const brandedImage = imageLocator
+    ? getBrandedImage(imageLocator, service)
+    : null;
```

Comment:
When we implement this issue https://github.com/bbc/simorgh/issues/5158, we will want to use `CpsMetadata` but this behaviour will not match the PGL requirement:
>  og:image = Service logo

We will see this image:
![image](https://user-images.githubusercontent.com/9645462/73006439-917d8300-3e02-11ea-9f57-b92a19bf1b65.png)

Rather than this one:
![image](https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png)
from:
https://github.com/bbc/simorgh/blob/1ee5f4d983dc4ac95eef50433591a084ea5f1fcb/src/app/lib/config/services/pidgin.js#L24

@lizcameron would this be acceptable? 

This should not affect the merge of this for MAP pages btw!

Reply:
Hi @andrewscfc 
Yes, it's actually preferred to use the promo image rather than the service logo. Service logo was specified for consistency with MAPs and to keep the MVP simple, but it seems subsequent work has superseded this and made it easier for us to use the promo image too.
Qu: if branded promo image isn't available, do we fall back to the service logo?

Reply:
> Qu: if branded promo image isn't available, do we fall back to the service logo? 
@lizcameron we do fallback to the service logo, I guess this means there's no more work to do for the PGLs